src/main/scala/com/atomist/rug/test/gherkin/AbstractExecutableFeature.scala

```diff
@@ -45,6 +45,8 @@ abstract class AbstractExecutableFeature[W <: ScenarioWorld](
             runGiven(world, step)
           case "When " if !world.aborted =>
             runWhen(world, step)
+          case "Then " if step.getText == "the scenario aborted" =>
```

Comment:
I think it would be useful to disambiguate between different exceptions in the tests.

Is there a way we could pass in the exception classname to capture?
e.g. `the scenario aborted due to [ScriptException]`.

We could leave this for another enhancement though....